clang/lib/Sema/DeclSpec.cpp

```diff
@@ -317,7 +317,7 @@ void Declarator::setDecompositionBindings(
           new DecompositionDeclarator::Binding[Bindings.size()];
```

Comment:
Note I'm not super fan of the hidden vector implementation here.
It forces Binding to be default constructible.